phoenix-core/src/it/java/org/apache/phoenix/end2end/MaxLookbackIT.java

```diff
@@ -317,16 +300,16 @@ public void testRecentMaxVersionsNotCompactedAway() throws Exception {
             //after flush, check to make sure we can see all three versions at the appropriate times
             assertMultiVersionLookbacks(dataTableSelectSql, allValues, allSCNs);
             assertMultiVersionLookbacks(indexTableSelectSql, allValues, allSCNs);
-            majorCompact(dataTable, EnvironmentEdgeManager.currentTimeMillis());
-            majorCompact(indexTable, EnvironmentEdgeManager.currentTimeMillis());
+            majorCompact(dataTable);
+            majorCompact(indexTable);
             //after major compaction, check to make sure we can see all three versions
             // at the appropriate times
             assertMultiVersionLookbacks(dataTableSelectSql, allValues, allSCNs);
             assertMultiVersionLookbacks(indexTableSelectSql, allValues, allSCNs);
             injectEdge.incrementValue(MAX_LOOKBACK_AGE * 1000);
             long afterLookbackAgeSCN = EnvironmentEdgeManager.currentTimeMillis();
-            majorCompact(dataTable, afterLookbackAgeSCN);
-            majorCompact(indexTable, afterLookbackAgeSCN);
+            majorCompact(dataTable);
+            majorCompact(indexTable);
```

Comment:
In the tests above, major compactions run on both the data and index table. Can we pick only one table for one test, the other for another test, and so on? The interesting problems arise when compaction runs on one table but not on both

Reply:
First, this test suite only was modified because I needed to steal its major compaction code and put it into TestUtil. It's otherwise unrelated to this JIRA, and if it needs changes, that should be a separate issue. 

Second, this isn't really a test of indexing per se, but of whether the max lookback coproc hooks correctly prevent data from being purged during the lookback window.

Reply:
But if there are additional indexing tests we need, definitely please suggest them in a JIRA!

Reply:
That is fair. Thanks.